core/json/src/main/scala/net/liftweb/json/Formats.scala

```diff
@@ -32,6 +32,14 @@ trait Formats { self: Formats =>
   val fieldSerializers: List[(Class[_], FieldSerializer[_])] = Nil
 
   /**
+   * Support for the experimental tuple decomposition/extraction that represents tuples as JSON
+   * arrays. This provides better support for heterogenous arrays in JSON, but enable it at your
+   * own risk as it does change the behavior of serialization/deserialization and comes
+   * with some caveats (such as Scala primitives not being recognized reliably during extraction).
```

Comment:
I think there should be a yellow flag of sorts that alerts someone who sees nothing except this flag being used that they should tread carefully. If you'd like to create an alias value named "serializeTuplesAsArrays" that reads from this and reference that directly in the extractor, that seems sensible to me. But at least until we're getting some positive reports from the field on this feature I'd like to have a warning inherent in the name. And calling it experimental fits the bill, imo.